lib/screens/config/v2/shuttle_bus_info.ex

```diff
@@ -0,0 +1,42 @@
+defmodule Screens.Config.V2.ShuttleBusInfo do
+  @moduledoc false
+
+  alias Screens.V2.WidgetInstance
+
+  @type t :: %__MODULE__{
+          minutes_range_to_destination: String.t(),
+          destination: String.t(),
+          arrow: arrow(),
+          english_boarding_instructions: String.t(),
+          spanish_boarding_instructions: String.t(),
+          priority: WidgetInstance.priority()
+        }
+
+  @type arrow :: :n | :ne | :e | :se | :s | :sw | :w | :nw | nil
+
+  @enforce_keys [
+    :minutes_range_to_destination,
+    :destination,
+    :arrow,
+    :english_boarding_instructions,
+    :spanish_boarding_instructions,
+    :priority
+  ]
+  defstruct minutes_range_to_destination: nil,
+            destination: nil,
+            arrow: nil,
+            english_boarding_instructions: nil,
+            spanish_boarding_instructions: nil,
+            priority: nil
+
+  use Screens.Config.Struct
+
+  for arrow <- ~w[n ne e se s sw w nw nil]a do
```

Comment:
`nil` will be handled correctly (`Jason` decodes JSON `null` as `nil`) so you don't need to add an explicit `value_from_json` clause for that one. Currently it would cause `{"arrow": "nil"}` to be converted to `%{arrow: nil}`.
```suggestion
  for arrow <- ~w[n ne e se s sw w nw]a do
```